pymongo/topology.py

```diff
@@ -57,8 +70,15 @@
     updated_topology_description,
 )
 
+if TYPE_CHECKING:
+    from bson import ObjectId
+    from pymongo.settings import TopologySettings
+    from pymongo.typings import _Address
+
+ClusterTime = Optional[Mapping[str, Any]]
 
-def process_events_queue(queue_ref):
+
+def process_events_queue(queue_ref: weakref.ReferenceType[Optional[queue.Queue]]) -> bool:
```

Comment:
Ditch the optional here as well.

Reply:
right, sorry for missing it, thanks for catching that!